lib/sla.js

```diff
@@ -1,6 +1,77 @@
+/*jslint node: true */
 'use strict';
 
-module.exports = function(callback){
-    console.log("\"snarflegluff\" should be printed below:");
-    var str = "snarflegluff";
-    return str;}
+/**
+ * Receives initial input and begins the application.
+ * @param {Object} client - Client received from app.js.
+ * @callback end - Handles errors and ends the application.
+ */
+module.exports = function(client, end) {
+  designateMixingBridge();
+
+  /**
+   * Looks for an existing mixing bridge and creates one if none exist.
+   */
+  function designateMixingBridge() {
```

Comment:
Because you are defining these functions inside the module.exports function, they will be created every time require is called on this file. You should move them above the export statement so they only need to be created once.